src/components/collection-progress-group/helpers/parse-collection-progress.ts

```diff
@@ -116,6 +116,29 @@ function getNextTutorialCta({
 	}
 }
 
+/**
+ * Given an array of tutorials, and an array of progressData,
+ * as well as a progressMatchFn that runs against the matched
+ * progressData record for each tutorial (if there is one),
+ *
+ * Return the first tutorial that meets the progressMatchFn criteria.
+ *
+ * Note that for tutorials without matched progressData,
+ * progressMatchFn() will be called with `undefined`.
+ */
+function findTutorialByProgress(
```

Comment:
Set up this helper to make it cleaner to find `"in_progress"` and "not-yet-in-progress" tutorials in separate steps (since if we can find the former, we don't need to look for the latter).